Refactor serialization getter/setters into torch.utils.serialization.config (#143324)
pytorch/pytorch: docs/source/notes/serialization.rst

.. autofunction:: get_unsafe_globals_in_checkpoint
.. autoclass:: safe_globals
.. autoclass:: skip_data

.. _serialization config:

Config
------

.. py:module:: torch.utils.serialization
.. py:module:: torch.utils.serialization.config

``torch.utils.serialization.config`` provides a global config that can control the behavior of
``torch.save`` and ``torch.load``.

``torch.utils.serialization.config.save`` contains options that control the behavior of ``torch.save``.

>   * ``compute_crc32``: whether to compute and write crc32 for each storage (Default : ``True``).

Reviewer: ```suggestion
  * ``compute_crc32``: whether to compute and write the zip file checksum (Default : ``True``).
```
Would this be more accurate? Isn't this skip writing the global checksum, not just the storage one?
